### ISSUE TITLE
Missing closing tag

### ISSUE THREAD

**Issue by stephlm2dev** [bradmartin/nativescript-checkbox#45]
In the following snippet (https://github.com/bradmartin/nativescript-checkbox/blob/master/README.md#angular-usage-sample), the third closing tag of `<Button>` is missing 

`</Button>` instead of `</`

```
<StackLayout>
    <CheckBox #CB1 text="CheckBox Label" checked="false"></CheckBox>
    <Button (tap)="toggleCheck()" text="Toggle it!"></Button>
    <Button (tap)="getCheckProp()" text="Check Property"></
</StackLayout>
```

**Comment by EddyVerbruggen** [comment]
Can you please do a little PR for that?

**Comment by stephlm2dev** [comment]
Yes, sure.

**Comment by EddyVerbruggen** [comment]
Closed by #46
